src/main/webapp/index.html

```diff
@@ -1,12 +1,93 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <meta charset="UTF-8">
-    <title>Nina's Portfolio</title>
-    <link rel="stylesheet" href="style.css">
+    <meta charset="UTF-8" />
+    <title>School Form</title>
+    <link rel="stylesheet" href="style.css" />
     <script src="script.js"></script>
   </head>
   <body>
-      <p> Hello </p>
+    <p>Rate your classes and professors!</p>
+    <form action="/data" method="POST">
+      <p>Rate your class:</p>
+      <div id="class-id"></div>
+      <button onclick="getClassID()">Get class ID</button>
+      <br /><br />
```

Comment:
Nit: since this is a prototype it's alright but generally multiple <br> tags mean you should be using css to do what you're doing instead

Reply:
Ohh you mean the line breaks?

Reply:
Oh lol, typing `<br>` actually inserts a break : P Yeah those